src/core/lib/channel/http_server_filter.c

```diff
@@ -314,13 +314,17 @@ static grpc_error *init_call_elem(grpc_exec_ctx *exec_ctx,
   grpc_closure_init(&calld->hs_on_recv, hs_on_recv, elem);
   grpc_closure_init(&calld->hs_on_complete, hs_on_complete, elem);
   grpc_closure_init(&calld->hs_recv_message_ready, hs_recv_message_ready, elem);
+  gpr_slice_buffer_init(&calld->read_slice_buffer);
```

Comment:
interesting. Was this motivated by a fuzzer failure? While a good practice to initialize and dispose of anything in calld, I wonder which path would end up using an uninitialized `calld->read_slice_buffer`. It seems the use around line 165 is already covered by another init.